analyzers/its/regression-test.ps1

```diff
@@ -33,6 +33,9 @@ function Prepare-Project([string]$ProjectName){
     $Output = ".\output\$ProjectName"
     New-Item -ItemType directory -Path $Output | out-null
 
+    Write-Output "Copying `Directory.Build.targets` to the project directory ($ProjectName)..."
```

Comment:
nitpick: I don't get the `...` at the end, because they will be concatenated to the project name, with no space.

But I wouldn't rerun the full CI just for that.